baseband/gsb/base.py

```diff
@@ -415,10 +402,9 @@ class GSBStreamWriter(GSBStreamBase, VLBIStreamWriterBase):
         Redundant modulo-8 shared memory block number; not used by Baseband.
     """
 
-    def __init__(self, fh_ts, fh_raw, header=None, nchan=None, bps=None,
+    def __init__(self, fh_ts, fh_raw, sample_rate, nchan=None, bps=None,
```

Comment:
`header` should stay up front (after `sample_rate`)

Reply:
For Mark 5 and VDIF, `header` is slotted just before `squeeze`.  I'll move it back to its previous position in GSB, and make a note in the issue we raised about this.

Reply:
OK, sounds good. There is some sense in having `header` (for the writer) right after all the things that cannot be inferred from it, and before the keywords that might define it.